src/include/distributed/multi_physical_planner.h

```diff
@@ -127,7 +127,7 @@ typedef struct Job
 	bool subqueryPushdown;
 	bool requiresMasterEvaluation; /* only applies to modify jobs */
 	bool deferredPruning;
-	Const *partitionValueConst;
+	Const *partitionKeyConst;
```

Comment:
I'm a bit confused about this change. The `Const` represents a value, not a key, right?

Reply:
yes it is a value.  I am making a naming change in the api. going for partition_key instead of partition_value. Here making it partitionKeyValueConst might be better for readability